database/migrations/migrations.go

```diff
@@ -75,6 +84,32 @@ func (e EmbedFSWrapper) Open(name string) (fs.File, error) {
 
 	s := string(b)
 
+	privateKey, err := crypto.GenerateKey()
+	if err != nil {
+		return nil, err
+	}
+
+	publicKey := privateKey.Public()
+	publicKeyECDSA, ok := publicKey.(*ecdsa.PublicKey)
+	if !ok {
+		return nil, errors.New("Unable to parse public key")
+	}
+
+	publicKeyBytes := crypto.FromECDSAPub(publicKeyECDSA)
+	privateKeyBytes := crypto.FromECDSA(privateKey)
+
+	keyPair := NodeKeyPair{
+		PublicKey:  hexutil.Encode(publicKeyBytes)[4:],
+		PrivateKey: hexutil.Encode(privateKeyBytes)[2:],
+	}
+
+	keyPairJson, err := json.Marshal(keyPair)
+	if err != nil {
+		return nil, err
+	}
+
+	constants["NODE_KEY"] = string(keyPairJson)
```

Comment:
A database migration is missing that actually sets this `{{NODE_KEY}}` in  node_attribute (similar to 000004_seed_node_attribute.up.sql)